monkey/monkey_island/cc/ui/src/components/ui-components/credential-pairs/CredentialPairs.js

```diff
@@ -205,20 +205,16 @@ const CredentialPairs = (props) => {
 
   return (
     <div id="configure-propagation-credentials">
+      <Typography variant="h5" component="h5">Credentials</Typography>
```

Comment:
I don't think this is needed if we have "Configured Credentials" before the table.